src/soc/samsung/exynos5422.c

```diff
@@ -289,7 +289,7 @@ static int exynos5422Setup(void) {
 	 * Mapping the GPIO register areas.
 	 * -> 0: 0x13400000, 1: 0x13410000, 2: 0x14000000, 3: 0x14010000, 4: 0x03860000
 	 */
-	for(i = 0; i < 5; ++i) {
+	for (i = 0; i < 5; ++i) {
```

Comment:
Oeps 😄

Reply:
[Here](https://github.com/wiringX/wiringX/pull/114#discussion_r1008352280) you asked to move them to a different PR, so I reverted them.

Reply:
You're right. Didn't saw the other filename.